ticdc/ticdc-sink-to-pulsar.md

```diff
@@ -0,0 +1,298 @@
+    auth-tls-certificate-path="/data/pulsar/certificate"
+    # Private key path of the Pulsar TLS encrypted authentication
+    auth-tls-private-key-path="/data/pulsar/certificate.key"
+    # Path to trusted certificate file of the Pulsar TLS encrypted authentication
+    tls-trust-certs-file-path="/data/pulsar/tls-trust-certs-file"
+    ```
+
+- OAuth2 authentication
+
+    Sink URI: 
+
+    ```shell
+    --sink-uri="pulsar+ssl://127.0.0.1:6650/persistent://public/default/yktest?protocol=canal-json"
+    ```
+
+    Config parameters: 
+
+    ```toml
+    [sink.pulsar-config]
+    # Pulsar oauth2 issuer-url. For more information, see the Pulsar website: https://pulsar.apache.org/docs/2.10.x/client-libraries-go/#oauth2-authentication
+    oauth2.oauth2-issuer-url="https://xxxx.auth0.com"
+    # Pulsar oauth2 audience
+    oauth2.oauth2-audience="https://xxxx.auth0.com/api/v2/"
+    # Pulsar oauth2 private-key
+    oauth2.oauth2-private-key="/data/pulsar/privateKey"
+    # Pulsar oauth2 client-id
+    oauth2.oauth2-client-id="0Xx...Yyxeny"
+    # Pulsar oauth2 oauth2-scope
+    oauth2.oauth2-scope="xxxx"
+    ```
+
+## Customize the dispatching rules for topics and partitions in Pulsar Sink
+
+### Matching rules for Matcher
+
+Take the `dispatchers` configuration item in the following sample configuration file as an example:
+
+```toml
+[sink]
+dispatchers = [
+  {matcher = ['test1.*', 'test2.*'], topic = "Topic expression 1", partition = "ts" },
+  {matcher = ['test3.*', 'test4.*'], topic = "Topic expression 2", partition = "index-value" },
+  {matcher = ['test1.*', 'test5.*'], topic = "Topic expression 3", partition = "table"},
+  {matcher = ['test6.*'], partition = "default"},
+  {matcher = ['test7.*'], partition = "test123"}
+]
+```
+
+- The tables that match the matcher rule are dispatched according to the policy specified by the corresponding topic expression. For example, the table `test3.aa` is dispatched according to `Topic expression 2`, and the table `test5.aa` is dispatched according to `Topic expression 3`.
+- For a table that matches more than one matcher rule, the topic expression corresponding to the top matcher will take precedence. For example, the table `test1.aa` is dispatched according to `Topic expression 1`.
+- For tables that do not match any matcher, the corresponding data change events are sent to the default topic specified in `-sink-uri`. For example, the table `test10.aa` is sent to the default topic.
+- For tables that match the matcher rule but do not have a topic dispatcher specified, the corresponding data changes are sent to the default topic specified in `-sink-uri`. For example, the table `test6.abc` is sent to the default topic.
+
+### Topic dispatcher
+
+A topic dispatcher is specified with `topic = "xxx"` and uses topic expressions to implement a flexible topic dispatching policy. It is recommended that the total number of topics be less than 1000.
```

Comment:
```suggestion
You can use `topic = "xxx"` to specify a topic dispatcher and use topic expressions to implement flexible topic dispatching policies. It is recommended that the total number of topics be less than 1000.
```

Reply:
align with the translation in https://docs.pingcap.com/tidb/dev/ticdc-sink-to-kafka#topic-dispatchers